### PR TITLE
Added the extensibility for the custom text block, Image and ActionButton component

### PR DESCRIPTION
**Pull request by IbrahimSulai**
Added the extensibility for the custom text block component

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/5275)

### CODE REVIEW

**Review by vivekvijayakrishnan — APPROVED** [review]
LGTM

File: source/community/reactnative/src/visualizer/renderer.js [hunk 1]
```diff
@@ -169,7 +181,7 @@ export default class Renderer extends React.Component {
                 ],
                 { cancelable: false }
             )
-        } else if (actionObject.type === "Action.OpenUrl" && !Utils.isNullOrEmpty(actionObject.url)) {
+        } else if (actionObject.type === "Action.OpenUrl" && !Utils.isNullOrEmpty(actionObject.url) || actionObject.type === "CustomActionButton") {
```
Comment (vivekvijayakrishnan) [inline source/community/reactnative/src/visualizer/renderer.js:184]: `CustomActionButton` can be anything such as `URL / submit or showCard`. we can just alert it with the payload details. Please use **_`alertAction`_** method. I think we are already calling **_`alertAction`_** by default if it's not `Submit/URL`. So you can remove this condition.

**Review by vivekvijayakrishnan — CHANGES_REQUESTED** [review]

**Review by vivekvijayakrishnan — APPROVED** [review]
LGTM

File: source/community/reactnative/src/visualizer/renderer.js (hunk 1 above)
Comment (IbrahimSulai) [reply source/community/reactnative/src/visualizer/renderer.js]: Resolved

**Review by pragadeeshk — APPROVED** [review]
LGTM

**Pull request merged**